cypress/e2e/donor/certificate-provider.cy.js

```diff
@@ -64,7 +64,7 @@ describe('Certificate provider task', () => {
         cy.checkA11yApp({ rules: { 'aria-allowed-attr': { enabled: false } } });
 
         cy.contains('label', 'Online and by email').click();
-        cy.get('#f-email').type(TestEmail);
+        cy.get('#f-email').type(TestEmail, { force: true });
```

Comment:
this kept failing, but also I think from what has been said will be redesigned at some point, so just lazily added `force` to get it working consistently